lib/build.js

```diff
@@ -48,7 +47,8 @@ function generateAssetsSync ({ verbose } = {}) {
 
 function clean () {
   // doesn't clean extensions.scss, should it?
-  del.sync(['public/**', '.port.tmp', portTmpFile])
+  // TODO: These files are deprecated, remove this line in v14?
+  del.sync(['public/**', '.port.tmp', '.tmp/port.tmp'])
```

Comment:
why do we still need to delete the port files?

Reply:
Just in case the files are still around for some reason. It's not strictly necessary, just being neat.